packages/iframe-coordinator/src/elements/frame-router.ts

```diff
@@ -152,6 +152,26 @@ export default class FrameRouterElement extends HTMLElement {
     }
   }
 
+  /**
+   * @property {object} frameSetup A property that can be set to initialize the host frame with the
```

Comment:
I think I'm missing something, because I'm getting a lint error `Documentation must exist for properties.` Any idea what to add here?

Reply:
This is what I would do (that seems to address the linting concerns). I would create and interface for this object (and include the JSDoc comments there) and then add a simplified JSDoc comment on either the getter or setter.

Maybe something like this:
`/** A property that can be set to initialize the host frame with the
 * possible clients and the environmental data required by the clients
 */
interface FrameSetup {
  /** The map of registrations for the available clients. */
  clients: RoutingMap;
  /** Information about the host environment. */
  envData: EnvData;
}`

` private _frameSetup: FrameSetup;
  /** Additional Comments Here? */
  get frameSetup(): any {
    return this._frameSetup;
  }

  set frameSetup(frameSetup: FrameSetup) {
    this._frameSetup = frameSetup;
    this._setSetupFrames(frameSetup.clients, frameSetup.envData);
  }`